### PR TITLE
Add prose style to Markdown content

### PR DESCRIPTION
**Pull request by CharlyMartin**
This PR extends the `prose` base style and adds a custom style based on the design team's recommendations.

Before merging, the dummy article, `test-for-design-team`, needs to be deleted.

---

Reference: https://github.com/tailwindlabs/tailwindcss-typography?tab=readme-ov-file#adding-custom-color-themes

Related to https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/67

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -15,6 +15,56 @@ module.exports = {
...
+            '--tw-prose-bullets': theme('colors.brand.100'),
+            '--tw-prose-code': theme('colors.brand.100'),
+            '--tw-prose-pre-bg': theme('colors.brand.700'),
+            '--tw-prose-pre-code': theme('colors.brand.300'),
+
+            a: {
+              textDecoration: 'none',
+              '&:hover': {
+                textDecoration: 'underline',
+              },
```
Comment (mirhamasala) [inline tailwind.config.js:37]: What about `focus` styles?

File: tailwind.config.js [hunk 2]
```diff
@@ -15,6 +15,56 @@ module.exports = {
...
+            },
+
+            code: {
+              '&::before, &::after': {
+                content: 'none !important',
+              },
+            },
```
Comment (mirhamasala) [inline tailwind.config.js:44]: What does this do?

File: tailwind.config.js [hunk 3]
```diff
@@ -15,6 +15,56 @@ module.exports = {
...
+            iframe: {
+              width: '100%',
+
+              '&::focus-visible': {
+                outline: 'none',
+              },
+            },
+
+            'iframe[src*="youtube.com"]': {
+              width: '100%',
```
Comment (mirhamasala) [inline tailwind.config.js:61]: Isn't this already set on `iframe`?

File: tailwind.config.js [hunk 4]
```diff
@@ -15,6 +15,56 @@ module.exports = {
...
+              borderStyle: 'solid',
+              borderColor: theme('colors.brand.600'),
+            },
+
+            iframe: {
+              width: '100%',
+
+              '&::focus-visible': {
+                outline: 'none',
+              },
```
Comment (mirhamasala) [inline tailwind.config.js:57]: Also - Do we not want these styles to apply to all iframes?

Comment (mirhamasala) [inline tailwind.config.js:57]: What does this do? (And if this is necessary, remember that by setting `ouline: 'none'` we probably just messed with a11y.

**Review by mirhamasala — CHANGES_REQUESTED** [review]
Few minor questions / comments - but looking good. 🤩 🤑

File: tailwind.config.js (hunk 1 above)
Comment (CharlyMartin) [reply tailwind.config.js:37]: Good catch

File: tailwind.config.js (hunk 2 above)
Comment (CharlyMartin) [reply tailwind.config.js:44]: This removes the visible backticks from the inline code style.

**default**
CleanShot 2024-05-02 at 10 56 37

**now**
CleanShot 2024-05-02 at 10 57 08

**Review by mirhamasala — APPROVED** [review]

**Pull request merged**